detection_rules/config.py

```diff
@@ -196,6 +219,11 @@ def parse_rules_config(path: Optional[Path] = None) -> RulesConfig:
     # paths are relative
     files = {f'{k}_file': base_dir.joinpath(v) for k, v in loaded['files'].items()}
     contents = {k: load_dump(str(base_dir.joinpath(v))) for k, v in loaded['files'].items()}
+
+    # check required package fields
+    if 'packages' in contents and 'package' in contents['packages']:
```

Comment:
based on this [comment](https://github.com/elastic/detection-rules/pull/3653/files#r1594288060), you can probably delete `check_path` and `validate_package_contents` and just do the check for `package.name`

Reply:
If we want to support people managing schemas then they need the `kibana.version`.

Reply:
name is version

Reply:
Oh right! so maybe we can simplify this.